### ISSUE TITLE
Noise simulations will have discontinuities at data distribution boundaries.

### ISSUE THREAD

**Issue by tskisner** [hpc4cmb/toast#7]
Since current noise sims have each process simulate a piece of timestream independently, there will be discontinuities at the process boundaries.  The solution is to either redistribute the data so that one process has the whole timestream, OR have every process simulate the whole dataset up to their local start sample, OR to use a python wrapper around fftw-mpi.

**Comment by tskisner** [comment]
For short noise simulations, we have worked around this by distributing over detector.  Simultaneous distribution by both time and detector was added in 5568fa457af4efcdc649b167db4405040e255f11.  The longer-term solution to support very long noise simulations will likely be one of:

- A hybrid simulation which generates a realization of low-frequency modes separate from the usual noise PSD.

- The use of fftw-mpi to generate a single long detector realization using multiple processes.
